src/controllers/pokestop.js

```diff
@@ -1,6 +1,7 @@
 const geoTz = require('geo-tz')
 const moment = require('moment-timezone')
 const Controller = require('./controller')
+const { log } = require('../lib/logger')
```

Comment:
you're requiring logger, but not using it

Reply:
Remains from debugging, will remove